common/src/chain/config/builder.rs

```diff
@@ -526,6 +545,11 @@ impl Builder {
     builder_method!(epoch_length: NonZeroU64);
     builder_method!(sealed_epoch_distance_from_tip: usize);
 
+    pub fn checkpoints(mut self, checkpoints: BTreeMap<BlockHeight, Id<GenBlock>>) -> Self {
+        self.checkpoints = Some(checkpoints);
+        self
+    }
```

Comment:
Then if genesis is included by default, you can use the default builder_method macro.